# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/xjdic23/no-random-return-value.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


The package failed on PowerPC since it returned a random value in main.

  - Rene Rebe <rene@rocklinux.org>

--- ./xjdxgen.c.orig	2003-10-01 09:18:58.000000000 +0200
+++ ./xjdxgen.c	2003-10-01 09:47:06.000000000 +0200
@@ -273,6 +273,7 @@
   jindex[0] = diclen+jiver;
   fwrite(jindex,sizeof(long),indptr+1,fp);
   fclose(fp);
+  exit(0);
 }
 /*======function to sort jindex table====================*/
 void jqsort(long lhs, long rhs)
